packages/trigger/src/lib/helpers.ts

```
@@ -0,0 +1,210 @@
+	if (!value || value === "-") return null;
+
+	const cleaned = value.replace(/,/g, "").trim();
+	const num = Number.parseInt(cleaned, 10);
+
+	return Number.isNaN(num) ? null : num;
+}
+
+/**
+ * Parse float from string, return null if invalid
+ */
+export function parseFloat(value: string | null): number | null {
```

Biome (2.1.2)

[error] Do not shadow the global "parseFloat" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)